tasks/validate/validate.yml

```diff
@@ -2,10 +2,10 @@
 - name: Check whether you are using a supported NGINX distribution
   ansible.builtin.assert:
     that:
-      - "{{ ansible_facts['distribution'] | lower in nginx_distributions.keys() | list }}"
-      - "{{ (ansible_facts['distribution_version'] | regex_search('\\d{1,2}\\.\\d{2}') | float in nginx_distributions[ansible_facts['distribution'] | lower]['versions'] | map('float'))
-        if ansible_facts['distribution'] | lower in ['alpine', 'ubuntu'] else ansible_facts['distribution_major_version'] in nginx_distributions[ansible_facts['distribution'] | lower]['versions'] | string }}"
-      - "{{ ansible_facts['architecture'] in nginx_distributions[ansible_facts['distribution'] | lower]['architectures'] }}"
+      - ansible_facts['distribution'] | lower in nginx_distributions.keys() | list
+      - (ansible_facts['distribution_version'] | regex_search('\\d{1,2}\\.\\d{2}') | float in nginx_distributions[ansible_facts['distribution'] | lower]['versions'] | map('float')
```

Comment:
Can you move the parentheses back to where they were? I've run into situations using the "X if Y" conditional in Ansible has led to issues when the "X" was not wrapped in parenthese :)